plutus-tx/src/PlutusTx/Builtins/Internal.hs

```diff
@@ -21,6 +21,12 @@
   Please note that the documentation for each function will only include operational invariants
   if there are any. This documentation assumes type system correctly enforces and prevents any structural
   errors on the generated UPLC. See Note [Structural vs operational errors within builtins].
+
```

Comment:
A couple of lines above it says "assumes type system". That should probably be "assumes that the type system".